cli/start.go

```diff
@@ -175,7 +175,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	log.Flush()
 }
 
-// A exterminateCmd command shuts down the node server.
+// An exterminateCmd command shuts down the node server.
```

Comment:
you can remove "A/An" entirely, comments should start with the variable/function name.